config/routes.rb

```diff
@@ -53,6 +53,7 @@
       resources :organization_memberships, only: %i[update destroy create]
       resources :permissions, only: %i[index]
       resources :reactions, only: [:update]
+      resources :push_notification_targets, only: %i[index new create edit update destroy]
```

Comment:
non-blocking: what about `/push_notifications/targets` ? This way if we need to add `/devices` too, they can leave under the same namesppace

Reply:
Same as in another comment, the naming is very tricky. After a refactor I settled for `AppIntegration` and I think it works quite well with the idea of nesting `/devices` because they are referencing App Integrations (i.e. `/admin/app_integrations/3/devices`).

Again, the conversation also [continues here](https://github.com/forem/forem/pull/13304#discussion_r611375160)